tests/DCPS/Compiler/annotations/classic_langmap/topic_annotations_test_lib/.gitignore

```diff
@@ -0,0 +1,2 @@
+/topic_annotations_test*
```

Comment:
This could be an issue when someone tries to add a file that happens to match the pattern.  I would prefer to have the .gitignore files generated by the script.  If there are reasons not to do that (or if the script needs to be changed), we can discuss other approaches.

Reply:
The thing I don't like about the script is that it relies getting the correct files indexed before running it, so it's not really very different from manually creating the gitignore. Would it be possible to have MPC generate .gitignore files?